test/conformance/configuration_test.go

```diff
@@ -34,8 +35,8 @@ func TestUpdateConfigurationMetadata(t *testing.T) {
 	logger := logging.GetContextLogger("TestUpdateConfigurationMetadata")
 
 	var names test.ResourceNames
-	names.Service = test.AppendRandomString("test-update-configuration-meta-", logger)
-	names.Config = names.Service
+	names.Config = test.AppendRandomString("test-update-configuration-meta-", logger)
```

Comment:
are those intentionally different?

Reply:
If your question is about Service -> Config, this test never used Service so that was not necessary. 
If you question is about generating a new random suffix for Route instead of using the same suffix as Config, I figured that it did not matter either way so followed the existing behavior of TestRouteCreation.